### PR TITLE
Typo "Azure file storage"→"Azure File Storage"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/azure/architecture/example-scenario/file-storage/moodle-azure-netapp-files
https://github.com/MicrosoftDocs/architecture-center/blob/main/docs/example-scenario/file-storage/moodle-azure-netapp-files-content.md
#PingMSFTDocs

### CODE REVIEW

**Comment by Jak-MS** [comment]
@arntdg 

Can you review this PR?

Note that Microsoft branded products/services include Microsoft Azure Storage or Azure Storage ... (NOT Azure File Storage)... it's an umbrella term for Blob Storage, Azure Files, Queue Storage, and Table Storage. Thanks.

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by arntdg** [comment]
I'm confused. The article talks about 'Azure NetApp Files', which is a first party Microsoft service, and a such a Microsoft product. What is the issue, what should change in the article, kindly expand. Thank you.

**Comment by Jak-MS** [comment]
@arntdg - responded via email.

File: docs/example-scenario/file-storage/moodle-azure-netapp-files-content.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Moodle strives to meet the demands of home workers and to provide the best possi
 - Consistent high-throughput, low-latency access to shared storage.
 - A way to scale up the solution to accommodate an increasing number of concurrent users. Customers prefer autoscaling configurations.
 
-This article outlines a solution that meets Moodle's needs. At the core of the solution is Azure NetApp Files, a first-party Azure file storage service. You can use this service to migrate and run the most demanding enterprise-scale file workloads in the cloud:
+This article outlines a solution that meets Moodle's needs. At the core of the solution is Azure NetApp Files, a first-party Azure File Storage service. You can use this service to migrate and run the most demanding enterprise-scale file workloads in the cloud:
```
Comment (Jak-MS) [inline docs/example-scenario/file-storage/moodle-azure-netapp-files-content.md:10]:
```suggestion
This article outlines a solution that meets Moodle's needs. At the core of the solution is Azure NetApp Files, a first-party storage service. You can use this service to migrate and run the most demanding enterprise-scale file workloads in the cloud:
```
removing the confusion (as suggested by @arntdg )

Comment (hyoshioka0128) [reply docs/example-scenario/file-storage/moodle-azure-netapp-files-content.md]: @Jak-MS - Thanks!

**Comment by Jak-MS** [comment]
@arntdg - merging this now. thanks.

**Review by Jak-MS — APPROVED** [review]

**Pull request merged**